ci/infra/openstack/cloud-init/commands.tpl

```diff
@@ -1 +0,0 @@
-  - [ zypper, -n, install, ${packages} ]
```

Comment:
By removing this we are also removing the installation of the internal CA, that's the errors shown on CI currently.